toolkit/offchain/tests/docker/entrypoint.sh

```diff
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+chmod 400 /config/keys/kes.skey
+chmod 400 /config/keys/vrf.skey
+
```

Comment:
New image is already pushed. There was a problem with old image, because it was not enough to set 400 in my local repo.